.github/workflows/code-style.yaml

```diff
@@ -19,3 +19,18 @@ jobs:
         with:
```

Comment:
Yo borraría estos dos, ruff deberia ser drop in replacement de ambos

Reply:
Agree